### PR TITLE
Translations for mailers and views

### LINKED ISSUE #11: Support I18n
Hey man, 
I have been reading up on how to add I18n support to a gem, and mostly things seem pretty straightforward. 
The main issue I have come up with is how to provide an easy way for someone using the gem to customize the YAML files -  similar to how devise loads `devise.en.yml` into the main app's `config/locales` directory.

It seems like this happens in the gem's engine, but I haven't been able to find an example or documentation on how to do that. I have a feeling it is quite simple 😄 

Have you done something like this before? Do you know of a resource I can turn to? 

Thanks! I am looking forward to creating a PR for this, as soon as I sort out what steps are required

### PR DESCRIPTION
**Pull request by mftaff**
closes #11

### CODE REVIEW

File: app/mailers/passwordless/mailer.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def magic_link(session)
       email_field = @session.authenticatable.class.passwordless_email_field
       mail(
         to: @session.authenticatable.send(email_field),
-        subject: 'Your magic link ✨'
+        subject: I18n.t('passwordless.mailers.subject')
```
Comment (mikker) [inline app/mailers/passwordless/mailer.rb]: I think `passwordless.mailer.subject` (singular `mailer`) is closer to the actual structure. Also – we only have one :)

File: app/views/passwordless/sessions/create.html.erb [hunk 2]
```diff
@@ -1 +1 @@
-<p>If we found you in the system, we've sent you an email.</p>
+<p><%= I18n.t('passwordless.sessions.success.email_sent') %></p>
```
Comment (mikker) [inline app/views/passwordless/sessions/create.html.erb]: Let's do `lib.controller.action.key` everywhere. So `passwordless.sessions.create.email_sent_if_record_found` maybe?

File: app/views/passwordless/sessions/new.html.erb [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <%= form_for @session, url: send(authenticatable_resource).sign_in_path do |f| %>
   <% email_field_name = :"passwordless[#{@email_field}]" %>
   <%= text_field_tag email_field_name, params.fetch(email_field_name, nil) %>
-  <%= f.submit 'Send magic link' %>
+  <%= f.submit I18n.t('passwordless.sessions.new.submit') %>
```
Comment (mikker) [inline app/views/passwordless/sessions/new.html.erb]: Perfect 👌

File: config/locales/en.yml [hunk 4]
```diff
@@ -3,3 +3,10 @@ en:
   passwordless:
     sessions:
       session_expired: 'Your session has expired, please sign in again.'
```
Comment (mikker) [inline config/locales/en.yml]: This should follow same structure as the others, so throw it under `create`.

**Comment by mftaff** [comment]
Brilliant!

**Comment by mikker** [comment]
_You are!_ Thanks 💙💚💛💜❤️

**Pull request merged**